lib/config.js

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const config = require('bedrock').config;
+const path = require('path');
+
+config['authn-did-jwt'] = {};
+config['authn-did-jwt'].routes = {
+  login: '/authn/did-jwt/login',
+  token: '/authn/did-jwt/token'
+};
+
+// NOTE: config.jsonld.strictSSL from bedrock core should be configured
+// appropriately, the default is `false`
+config['authn-did-jwt'].didio = {
+  // FIXME: some computed config?
+  baseUrl: 'https://authorization.io/dids/'
+};
+
+config['authn-did-jwt'].namespaces = {
+  hmac: {
+    id: 'HMAC',
```

Comment:
We should identify the namespaces based on actions or purpose for the provisioned keys (e.g. "cross-domain-authn" vs. "same-domain-authn"), not algorithms.